modules/concepts/hooks/list-of-hooks/displayTopColumn.md

```diff
@@ -0,0 +1,27 @@
+---
+Title: displayTopColumn
```

Comment:
It's odd that there is no origin of this hook, it doesn't look like a dynamic one

Reply:
Seems like it was related to `homeslider` maybe an old module for 1.6

But the hook itself is not in the core anymore, this one is a remnant from the hook.xml file not a hook found by the extractor